### ISSUE TITLE
Loop check or wait to that timestamp?

### ISSUE THREAD

**Issue by Jack-Kingdom** [dbader/schedule#180]
How about we calculate the most recent job and wait to that point. ( thread sleep or sth etc.)
Looks like It's more effective than loop check.

**Comment by luckydonald** [comment]
Yeah, having a longer sleep would be better than having to wake from sleep every x seconds.
